src/ResourceManager/OperationalInsights/Commands.OperationalInsights/DataSources/GetAzureOperationalInsightsDataSourceCommand.cs

```diff
@@ -57,7 +57,6 @@ public class GetAzureOperationalInsightsDataSourceCommand : OperationalInsightsB
         HelpMessage = "The data source name.")]
         [Parameter(ParameterSetName = ByWorkspaceObjectByKind)]
         [ValidateSet(
-            PSDataSourceKinds.AzureAuditLog,
```

Comment:
@haitch should there be a value in the validate set for `AzureActivityLog` if it is replacing `AzureAuditLog`?